modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -7,32 +7,68 @@
 = Using Data Mover for CSI snapshots
 
 :FeatureName: Data Mover for CSI snapshots
-include::snippets/technology-preview.adoc[]
 
-The OADP 1.1.2 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.2 Data Mover solution uses the Restic option of VolSync.
+The OADP Data Mover enables customers to back up Container Storage Interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications, using CSI volume snapshots pulled from the object store if a failure, accidental deletion, or corruption of the cluster occurs.
+
+The Data Mover solution uses the Restic option of VolSync.
 
-[NOTE]
-=====
 Data Mover supports backup and restore of CSI volume snapshots only.
 
-Currently, Data Mover does not support Google Cloud Storage (GCS) buckets.
-=====
+In OADP 1.2.x Data Mover `VolumeSnapshotBackups` (VSBs) and `VolumeSnapshotRestores` (VSRs) are queued using the VolumeSnapshotMover (VSM). The VSM's performance is improved by specifying a concurrent number of VSBs and VSRs simultaneously `InProgress`. After all async plugin operations are completed, the backup is marked as completed.
```

Comment:
Do the _In OADP 1.2.x_ and OAPD 1.1.x_ phrases need a comma? (I tend to be a bit comma-happy, but it seems to read better in most places.) 

> OADP 1.2.x, Data Mover

Reply:
@mburke5678 I'm a big fan of commas too, in this case, it may confuse the reader to add commas.

Reply:
@mburke5678 
Thank you for all of your valuable feedback. I have implemented several of your suggestions.